src/pages/index.astro

```diff
@@ -1,11 +1,20 @@
 ---
 import Layout from "../layouts/Layout.astro";
 import "../index.scss";
+import Navigation from "../components/PageHeader.astro";
 ---
 
-<Layout title="My portfolio" description="Portfolio presents my current programming skills, knowledge and stack" >
+<Layout
+  title="My portfolio"
+  description="Portfolio presents my current programming skills, knowledge and stack"
+>
+  <header>
+    <div class="container flex justify-center">
+      <Navigation />
+    </div>
+  </header>
```

Comment:
This header and it's child should be inside the component

Reply:
Changed